dashboard/src/components/PageHeader/PageHeader.v2.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from "react";
+
+import "./PageHeader.v2.css";
+
+interface IPageHeaderProps {
+  children: Array<boolean | JSX.Element> | JSX.Element;
+}
+
+function PageHeader(props: IPageHeaderProps) {
+  return <header className="kubeapps-header clr-row">{props.children}</header>;
```

Comment:
For this, I'd keep `header` as a separate HTML tags:

```suggestion
  return <header className="kubeapps-header">
    <Row>
      {props.children}
    </Row>
  </header>;
```

Rows should be used as independent items to avoid collisions between the styles.